### PR TITLE
Internal improvement: Org scoping

### PR DESCRIPTION
**Pull request by eggplantzzz**
1. truffle-contract-sources --> @truffle/contract-sources
  2. truffle-contract --> @truffle/contract
  3. truffle-config --> @truffle/config
  4. truffle-artifactor --> @truffle/artifactor

### CODE REVIEW

File: scripts/ci.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ elif [ "$GETH" = true ]; then
   run_geth
   sleep 30
   lerna run --scope truffle test --stream -- --exit
-  lerna run --scope truffle-contract test --stream -- --exit
+  lerna run --scope contract test --stream -- --exit
```
Comment (CruzMolina) [inline scripts/ci.sh]:
```suggestion
  lerna run --scope @truffle/contract test --stream -- --exit
```

Comment (eggplantzzz) [reply scripts/ci.sh]: Is this not a reference to the directory and not the package?

Comment (gnidan) [reply scripts/ci.sh]: I think it's supposed to be a reference to the package (at least that's what the build failure seemed to suggest)

Comment (CruzMolina) [reply scripts/ci.sh]: I thought so! https://github.com/trufflesuite/truffle/pull/2350/checks?check_run_id=208414445#step:6:954

Comment (eggplantzzz) [reply scripts/ci.sh]: Changed to the package name.

**Pull request merged**